src/services/services.ts

```diff
@@ -2912,6 +2919,63 @@ module ts {
 
                 return filteredMembers;
             }
+
+            function isExternalModuleName(token: Node): boolean {
+                if (token && token.parent && token.kind === SyntaxKind.StringLiteral && isInStringOrRegularExpressionOrTemplateLiteral(token)) {
+                    return token.parent.kind === SyntaxKind.ImportDeclaration ||
+                        token.parent.kind === SyntaxKind.ExportDeclaration ||
+                        token.parent.kind === SyntaxKind.ExternalModuleReference;

```

Comment:
Be careful not to include:
```ts
export = "hello";
```
if we've decided to support that. Also, don't include:
```ts
export default "hello";
```